cmd/atc/handler.go

```
@@ -105,7 +105,9 @@ func Handler(client *k8s.Client, cache *wasm.ModuleCache, controllers *atc.Contr
 				continue
 			}
 
-			unstructured.SetNestedField(raw, originalStatus, "status")
+			if err := unstructured.SetNestedField(raw, originalStatus, "status"); err != nil {
```

to conserve previous behaviour it should ignore the error here using `_`